.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    machine: true
```

Comment:
You probably don't need to include `machine: true`, but use `docker:` section instead. More information and code is [here](https://circleci.com/docs/2.0/configuration-reference/#docker--machine-executor). This way CI will literally take few seconds :)

Also, there are official [terraform images](https://hub.docker.com/r/hashicorp/terraform/) you may consider using.